parsl/configs/improv.py

```diff
@@ -0,0 +1,33 @@
+from parsl.config import Config
+from parsl.executors import HighThroughputExecutor
+from parsl.launchers import MpiRunLauncher
+from parsl.providers import PBSProProvider
+
+config = Config(
+        executors=[
+            HighThroughputExecutor(
+                label="Improv_multinode",
+                max_workers_per_node=32,
+                provider=PBSProProvider(
+                    account="YOUR_ALLOCATION_ON_IMPROV",
+                    # PBS directives (header lines): for array jobs pass '-J' option
```

Comment:
i don't think we should be encouraging people to use array jobs. i've only ever seen one dubious need for them.

Reply:
Removed.